cli/azd/docs/adding-command.md

```diff
@@ -0,0 +1,211 @@
+# Adding an azd command
```

Comment:
Thank you so much for writing this document!

:heart::heart::heart:

Reply:
any time :)